app/templates/forms/user_register_form.html

```diff
@@ -1,59 +1,47 @@
 {% import "bootstrap/wtf.html" as wtf %}
 {% from "forms/_formhelper.html" import render_field %}
 
-<form class="row g-3" action="{{ url_for('user.register') }}" method="POST">
+<main>
+  <div class="container pt-5 action="{{ url_for('user.register') }}" >
   {{ form.hidden_tag() }}
```

Comment:
Dont remove the `form` element

Reply:
It is not removed it is included inside the main and div containers

Reply:
ah I see. However, include the attributes from before:
` action="{{ url_for('user.register') }}" method="POST"`